scale/scheduler/test/test_views.py

```diff
@@ -22,10 +22,21 @@ def setUp(self):
         django.setup()
         Scheduler.objects.create(id=1)
 
+    def test_invalid_version(self):
+        """Tests calling the scheduler view with an invalid REST API version"""
+
+        url = '/v1/scheduler/'
+        response = self.client.generic('GET', url)
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND, response.content)
+        
```

Comment:
Your tests look good, just please separate the `TestSchedulerView` class into two separate test classes, one for v5 and one for v6 and split the code accordingly. This makes it easier to find, copy, and remove versions of the API.